e2e/tests/workspace/workspace-clone.spec.ts

```diff
@@ -84,12 +85,21 @@ describe('Clone workspace', () => {
       // strips out dash from workspace name
       expect(workspaceDataUrl).toContain(cloneWorkspaceName.replace(/-/g, ''));
 
+      // starting a new incognito page
```

Comment:
We probably want an `IncognitoBrowserWindow` class. Though the additional complexity here could be considerable.